samples/cellular/lwm2m_client/overlay-aggressive-psm.conf

```diff
@@ -0,0 +1,20 @@
+# Sets the duration that the lwm2m engine will be polling for data after transmission before
+# the socket is closed.
+# Adjust so that we can detach from network in 10 seconds
+CONFIG_LWM2M_QUEUE_MODE_UPTIME=10
+
+# Configure PSM mode
+# Request periodic TAU of 3600 seconds (60 minutes)
+CONFIG_LTE_PSM_REQ_RPTAU="00000110"
```

Comment:
This should be extended as well, like we discussed.

Be agressive on power saving and ask for a 12h TAU.